### ISSUE TITLE
Check skipping of mutation with consequence variant: stop_retained_variant

### ISSUE THREAD

**Issue by FedericaBrando** [bbglab/boostdm-pipeline#20]
When exploring the cross validation tables (50 iteration table) - specifically TP53 BRCA - I found a mutation that is not annotated with any of the consequence type we expect:

| chr | pos | ref | alt | CLUSTL | HotMaps | smRegions | PhyloP | nmd | csqn_type_missense | csqn_type_nonsense | csqn_type_splicing | csqn_type_synonymous | driver | boostDM_score | 
|:---:|:---:|:---:|:---:|---|---|---|---|---|---|---|---|---|---|---|
|17 | 7669610 | C | T | 0.0 | 0.0 | 0 | 2.013672 | 1 | 0 | 0 | 0 | 0 | 0 | 0.028891

in the vep.tsv.gz this mutation is the following:
```tsv
❯ tabix containers/datasets_24/boostdm/shared/vep.tsv.gz 17:7669610-7669610
17	7669610	C	T	ENSG00000141510	ENST00000269305	Transcript	stop_retained_variant	1323	1181	394	*	tGa/tAa	-	LOW	-	-1	-	TP53	HGNC	HGNC:11998	YES	NM_000546.6	-	ENSP00000269305	11/11	-
```

The consequence type is `stop_retained_variant` which is from the synonymous family, since is replacing a stop codon to another stop codon [[source1](https://rgd.mcw.edu/rgdweb/ontology/view.html?acc_id=SO:0001567), [source2](http://www.sequenceontology.org/miso/current_svn/term/SO:0001567)].

since we do not any consequence type anotation for this substitution, we end up losing this substitution in the final in silico saturation output:
[![image](https://github.com/user-attachments/assets/c308b5ba-665d-41d2-8ab9-2f440084608b)](https://beta.intogen.org/boostdm/search?ttype=BRCA&gene=TP53)

### The question is the following: Is this the desired behaviour? Shall we include the stop_retained_variant in the aggregation dictionary mapped to the synonymous? 

pin @koszulordie @JoanEnn

**Comment by koszulordie** [comment]
Yes, we must include the Sequence Ontology term `stop_retained_variant` in the "synonymous" consequence type.
